### PR TITLE
V12

### PR DESCRIPTION
**Pull request by Eldinnie**
test PR for codacy

### CODE REVIEW

File: examples/echobot2.py [hunk 1]
```diff
@@ -29,30 +30,32 @@
...
+    logger.warning('Update "%s" caused error "%s"', update, context.error)
 
 
 def main():
     """Start the bot."""
-    # Create the EventHandler and pass it your bot's token.
-    updater = Updater("TOKEN")
+    # Create the Updater and pass it your bot's token.
+    # Make sure to set use_context=True to use the new context based callbacks
+    # Post version 12 this will no longer be necessary
```
Comment (jsmnbom) [inline examples/echobot2.py]: Version 13? surely applies elsewhere too

File: CHANGES.rst [hunk 2]
```diff
@@ -2,6 +2,25 @@
 Changes
 =======
 
+**2018-??-??**
+*Released 11.0.0*
```
Comment (jsmnbom) [inline CHANGES.rst]: V12

File: CHANGES.rst [hunk 3]
```diff
@@ -2,6 +2,25 @@
...
+
+Context based callbacks:
+See https://git.io/vp113 for help.
```
Comment (jsmnbom) [inline CHANGES.rst]: Not correct url anymore
applies elsewhere too

Comment (Eldinnie) [reply CHANGES.rst]: Fixed it in #1221

**Pull request closed without merging**